ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -308,7 +308,7 @@ def test_remote_server(self):
         This test installs IPA server instead of replica on
         system under test with the same realm and domain name.
         """
-        tasks.install_master(self.replicas[0], setup_dns=True, setup_kra=True)
+        tasks.install_master(self.replicas[0], setup_dns=False, setup_kra=True)
```

Comment:
This is a wrong approach.

The test is succeeding upstream without your patch because the machines are properly configured for DNS.

When the test is run downstream, the playbooks configure replicas[0] with a DNS server = master before launching the test. As a consequence, the installation of a new IPA server on replicas[0] is using master as DNS forwarder and sees that the DNS zone is already handled by master.
The right fix must be done downstream in the yaml file and avoid using master as DNS server for replicas[0].

Reply:
@flo-renaud agreed with your comment. Closing this PR.